controlplane/eks/api/v1alpha3/conversion.go

```diff
@@ -89,3 +89,13 @@ func Convert_v1alpha3_NetworkSpec_To_v1alpha4_NetworkSpec(in *infrav1alpha3.Netw
 func Convert_v1alpha4_NetworkSpec_To_v1alpha3_NetworkSpec(in *infrav1alpha4.NetworkSpec, out *infrav1alpha3.NetworkSpec, s apiconversion.Scope) error {
 	return infrav1alpha3.Convert_v1alpha4_NetworkSpec_To_v1alpha3_NetworkSpec(in, out, s)
 }
+
+// Convert_v1alpha3_Instance_To_v1alpha4_Instance is an autogenerated conversion function.
+func Convert_v1alpha3_Instance_To_v1alpha4_Instance(in *infrav1alpha3.Instance, out *infrav1alpha4.Instance, s apiconversion.Scope) error {
+	return infrav1alpha3.Convert_v1alpha3_Instance_To_v1alpha4_Instance(in, out, s)
+}
+
+// Convert_v1alpha4_Instance_To_v1alpha3_Instance is an autogenerated conversion function.
+func Convert_v1alpha4_Instance_To_v1alpha3_Instance(in *infrav1alpha4.Instance, out *infrav1alpha3.Instance, s apiconversion.Scope) error {
+	return infrav1alpha3.Convert_v1alpha4_Instance_To_v1alpha3_Instance(in, out, s)
+}
```

Comment:
Remove these as they're no longer needed. Generated files should revert to their original.